src/bio2bel/sources/biogrid.py

```diff
@@ -0,0 +1,265 @@
+# -*- coding: utf-8 -*-
+
+"""This script downloads and parses BioGRID data and maps the interaction types to BEL."""
+
+import logging
+from typing import Iterable, List
+
+import numpy as np
+import pandas as pd
+from protmapper.uniprot_client import get_mnemonic
+from tqdm import tqdm
+
+import pybel.dsl
+from bio2bel.utils import ensure_path
+from pybel import BELGraph
+
+
+SEP = '\t'
+BIOGRID = 'biogrid'
+SOURCE = 'source'
+TARGET = 'target'
+ALT_SOURCE_ID = 'alt_source_id'
+ALT_TARGET_ID = 'alt_target_id'
+RELATION = 'relation'
+PUBMED_ID = 'pubmed_id'
+UNIPROT = 'uniprot'
+EVIDENCE = 'From BioGRID'
+MODULE_NAME = 'biogrid'
+
+VERSION = '3.5.183'
+BASE_URL = 'https://downloads.thebiogrid.org/Download/BioGRID/Release-Archive'
+URL = f'{BASE_URL}/BIOGRID-{VERSION}/BIOGRID-ALL-{VERSION}.mitab.zip'
+
+#: Relationship types in BioGRID that map to BEL relation 'increases'
+BIOGRID_INCREASES_ACTIONS = {
+    'synthetic genetic interaction defined by inequality',
+    'additive genetic interaction defined by inequality',
+}
```

Comment:
i would argue that neither of these are increase interactions, and both of them need more logic than just mapping the string. the entity types need to be modified as well

Reply:
In the case of genetic interactions, we do not have any information about which target in the end in affected by the interaction of those two genes. What I would suggest is to term them all as associations and modify the target as the gene. Do you have an additional suggestion?

Reply:
yes that's what I was going for. 

A while back I wanted to work with UCSF to define a standard for synthetic lethal pairs. if you can come up with a couple examples, then we could propose that as a BEL standard (would be a cool thing to include in your thesis as well)

In BEL it could look something like
```
geneticInteraction(g(X), g(Y), ...) -> biologicalProcess(GO:0008219 ! "Cell Death")
```

where the `geneticInteraction()` could behave with similar semantics to lists like `complexAbundance` or `compositeAbundance`. These are called "reifed" nodes, which means they're a node that implicitly represents edges.

If you can find a specific compelling example from the literature then I could walk you through making a BEL enhancement proposal at `https://bep.bel.bio`

Reply:
Do you mean checking the pubmed article itself?

Reply:
start googling "synthetic lethal pairs" and find a nice example of the biology itself, maybe from a review